js/App.js

```diff
@@ -71,6 +71,16 @@ App.prototype.playNotificationSound = function() {
   this._notificationSound.play();
 },
 
+App.prototype.showOverlay = function() {
```

Comment:
Created showOverlay and hideOverlay functions on the main app instance. It will save us from having to query the DOM in all the different views to get #overlay. Also, if we ever change the implementation, we have a centralized place to do so and it probably wouldn't require any change to the places which call it.